practice-one/src/index.html

```diff
@@ -42,5 +44,72 @@
         </nav>
     </header>
     <!-- End Header -->
+    <!-- Start footer -->
+    <footer class="footer-section">
+        <div class="footer-container">
+            <div class="footer-item-top">
+                <div class="item-top">
+                    <h4 class="text-primary text-white text-title-footer">contact us</h4>
+                    <address class="contact-us">
+                        <p class="text-primary description text-white opacity break">2300 W Arbutus Dr Citrus Springs, FL 34434, USA</p>
+                        <div>  
+                            <div class="text-primary description text-white opacity">+235 793-891-7938</div>
+                            <div class="text-primary description text-white opacity">contact@hofmannui.net</div>
+                        </div>
+                    </address>
+                </div>
+                <div class="item-top">
+                    <h4 class=" text-primary text-white text-title-footer">follow us</h4>
+                    <div class="follow-us">
+                        <a href="#">
+                            <img class="opacity" src="assets/images/icons/Pinterest.svg" alt="Pinterest"/>
+                        </a>
+                        <a href="#">
+                            <img class="opacity" src="assets/images/icons/twitter.svg" alt="Twitter"/>
+                        </a>
+                        <a href="#">
+                            <img class="opacity" src="assets/images/icons/facebook.svg" alt="Facebook"/>
+                        </a>
+                    </div>
+                </div>
+                <div class="item-top">
+                    <h4 class="text-primary text-white text-title-footer">about us</h4>
+                    <p class="text-primary description text-white opacity break-about-us">SmartWeb was made to help you quickly create beautiful layouts of responsive websites.</p>
+                </div>  
+            </div>
+
+            <hr class="rule">
+
+            <div class="footer-item-bottom">
+                <div class="text-primary description text-white opacity">© 2023 SmartWeb</div>
+                <nav>
+                    <ul class="navbar-menu">
+                        <li>
+                            <a class="text-primary navbar-item text-white" href="#">HOME</a>                    
```

Comment:
Shouldn't uppercase for content in HTML

Reply:
I fix it
![image](https://github.com/HoangLiarPhuongLe/html_css_training/assets/110210274/ae40d1b0-f4d8-4d98-a81d-e5c923e4808c)